frontend/src/components/Board/Column/Column.tsx

```diff
@@ -130,13 +144,29 @@ const Column = React.memo<ColumMemoProps>(
                 {(provided) => (
                   <Container direction="column" elevation="2">
                     <Flex css={{ pt: '$20', px: '$20', pb: '$16' }} justify="between">
-                      <Flex>
+                      <Flex css={{ width: '100%' }}>
                         {hasAdminRole && isRegularBoard && (
                           <Flex {...providedColumn.dragHandleProps}>
                             <Icon name="arrange" size={24} />
                           </Flex>
                         )}
-                        <Title heading="4">{title}</Title>
+                        <TitleContainer
+                          css={{
+                            width: hasMoreThanThreeColumns ? '$130' : '$237',
+                          }}
+                        >
+                          {showTooltip ? (
+                            <Tooltip content={title}>
+                              <Title heading="4" ref={ref}>
+                                {title}
+                              </Title>
```

Comment:
```suggestion
                              <Title heading="4" ref={ref}>
                                {title}
                              </Title>
```

As a suggestion, you can extract this code into a new component that can be reusable when needed in this file